StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Extensions/ConfirmButton+Link.swift

```diff
@@ -12,12 +12,13 @@ extension ConfirmButton {
 
     static func makeLinkButton(
         callToAction: CallToActionType,
+        appearance: PaymentSheet.Appearance,
         compact: Bool = false,
         didTap: @escaping () -> Void
     ) -> ConfirmButton {
         let directionalLayoutMargins = compact ? LinkUI.compactButtonMargins : LinkUI.buttonMargins
 
-        var appearance = LinkUI.appearance
+        var appearance = appearance
         appearance.primaryButton.height = LinkUI.primaryButtonHeight(margins: directionalLayoutMargins)
```

Comment:
Note that in a future PR, this may need to be updated if we want to allow customization via `LinkAppearance`. This is hardcoded call this method, but we’re not currently displaying UI from Crypto Onramp, so I didn’t want to make an unwanted change ahead of being able to test it.